wsd/RequestDetails.hpp

```diff
@@ -133,6 +133,12 @@ class RequestDetails
     /// Decode and sanitize a URI.
     static Poco::URI sanitizeURI(const std::string& uri);
 
+    // if request uri is an alias, replace request uri host and port with
```

Comment:
Documentation comments (three ///) are used to describe functions. See other functions in this file and elsewhere.